react/features/overlay/components/native/PageReloadOverlay.js

```diff
@@ -79,15 +77,13 @@ class PageReloadOverlay extends AbstractPageReloadOverlay<Props> {
         const { message, timeLeft, title } = this.state;
 
         return (
-            <OverlayFrame>
-                <ConfirmDialog
-                    cancelLabel = 'dialog.Cancel'
-                    confirmLabel = 'dialog.rejoinNow'
-                    descriptionKey = { `${t(message, { seconds: timeLeft })}` }
-                    onCancel = { this._onCancel }
-                    onSubmit = { this._onReloadNow }
-                    title = { title } />
-            </OverlayFrame>
+            <ConfirmDialog
```

Comment:
Pl get rid of the overlay component altogether. This needs to be a standalone component.

Reply:
This extends an Abstract component. We need its state object. I don't think it's a wise move.